### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by KittNeKit**

### CODE REVIEW

**Review by Bohdan-Salamakha — CHANGES_REQUESTED** [review]
fix an issue after "python create_file.py -f file.txt"
image

**Review by EdAlekseiev — CHANGES_REQUESTED** [review]
It does not work for the next commands
1. python create_file.py -d dir1  -f file.txt 
2. python create_file.py -f file.txt -d dir1
Please make your code more flexible

File: app/create_file.py [hunk 1]
```diff
@@ -1 +1,44 @@
-# write your code here
+import sys
+import os
+import datetime
+
+
+# function, which create text file and fill it
+def file_write(*filename) -> None:
```
Comment (olena-hul) [inline app/create_file.py:7]: Fix flake8

**Review by olena-hul — CHANGES_REQUESTED** [review]

**Review by olena-hul — APPROVED** [review]

**Comment by olena-hul** [comment]
Good job!

**Pull request closed without merging**